<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EquipmentInfoDialog</class>
 <widget class="QDialog" name="EquipmentInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>仪器信息</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>341</width>
     <height>381</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>101</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="add">
      <property name="text">
       <string>增</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="delete_">
      <property name="text">
       <string>删</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="change">
      <property name="text">
       <string>改</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="search">
      <property name="text">
       <string>查</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>仪器编号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EquipNo"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>260</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>仪器名称</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EquipName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>  数     量</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="EquipAmt"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Submit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>360</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
